### PR TITLE
Rename XAppConnectionManager to AbacusConnectionManager

### PR DESCRIPTION
**Pull request by nambrot**
Also renames XappConnectionClient to AbacusConnectionClient

Code to generate messageWithProof is at https://github.com/abacus-network/abacus-monorepo/compare/nambrot/generate-message-with-proof?expand=1

### CODE REVIEW

File: typescript/deploy/cache/console-history.txt [hunk 1]
```diff
@@ -0,0 +1,66 @@
+u.recoverPublicKey(u.arrayify(preprendedHash), "0x2e89389e277dd4e185984c4e3fc22ac66f3be75009550c272c24c5782d1d84c82d15a927e85e67831144914caa6e495811d41c7857b2efdab4defb99217089211c")
```
Comment (tkporter) [inline typescript/deploy/cache/console-history.txt:1]: what's this file?

File: typescript/deploy/cache/solidity-files-cache.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (tkporter) [inline typescript/deploy/cache/solidity-files-cache.json:1]: also curious about this file

File: typescript/deploy/config/environments/local/core/contracts/alfajores.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (tkporter) [inline typescript/deploy/config/environments/local/core/contracts/alfajores.json:1]: and curious about all these new `typescript/deploy/config` files

File: typescript/deploy/local-checkpointer-alfajores.env [hunk 4]
```diff
@@ -0,0 +1,24 @@
+OPT_BASE_OUTBOX_CONNECTION_URL=http://localhost:8545
```
Comment (tkporter) [inline typescript/deploy/local-checkpointer-alfajores.env:1]: are these `typescript/deploy/*.env` files also intentional?

File: typescript/infra/src/governance/deploy.ts [hunk 5]
```diff
@@ -35,19 +35,22 @@ export class AbacusGovernanceDeployer extends AbacusRouterDeployer<
       new GovernanceRouter__factory(signer),
       upgradeBeaconController.address,
       [config.recoveryTimelock],
-      [xAppConnectionManager.address],
+      [abacusConnectionManager.address],
     );
 
     // Only transfer ownership if a new XCM was deployed.
```
Comment (tkporter) [inline typescript/infra/src/governance/deploy.ts:41]: nit: probably worth replacing any `XCM` references too

**Review by tkporter — APPROVED** [review]
Generally lgtm- curious about the new files & there may be a few other names worth changing

File: solidity/app/contracts/AbacusConnectionClient.sol [hunk 6]
```diff
@@ -35,73 +35,75 @@ abstract contract XAppConnectionClient is OwnableUpgradeable {
...
+        address _abacusConnectionManager
+    ) internal {
+        _setAbacusConnectionManager(_abacusConnectionManager);
         __Ownable_init();
     }
 
     // ============ External functions ============
 
     /**
      * @notice Modify the contract the xApp uses to validate Inbox contracts
```
[the remark this answers is not in the document]
Comment (nambrot) [reply solidity/app/contracts/AbacusConnectionClient.sol:48]: Yes indeed, wanted to do this in another PR to keep the diffs managable

File: typescript/deploy/cache/console-history.txt (hunk 1 above)
Comment (nambrot) [reply typescript/deploy/cache/console-history.txt]: accident

**Pull request merged**